### PR TITLE
[Merged by Bors] - Merge devnet 3

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

N/A

## Proposed Changes

Changes required for the `merge-devnet-3`. Added some more non substantive renames on top of @realbigsean 's commit. 
Note: this doesn't include the proposer boosting changes in kintsugi v3.

This devnet isn't running with the proposer boosting fork choice changes so if we are looking to merge https://github.com/sigp/lighthouse/pull/2822 into `unstable`, then I think we should just maintain this branch for the devnet temporarily.

### CODE REVIEW

**Comment by michaelsproul** [comment]
I actually think it would be fine to merge this. We can hold off merging proposer boosting a bit longer

File: beacon_node/execution_layer/src/engine_api/http.rs [hunk 1]
```diff
@@ -74,6 +74,8 @@ impl HttpJsonRpc {
             .json()
             .await?;
 
+        dbg!(&body);
```
Comment (paulhauner) [inline beacon_node/execution_layer/src/engine_api/http.rs]:
```suggestion
```

File: beacon_node/execution_layer/src/engine_api/http.rs [hunk 2]
```diff
@@ -74,6 +74,8 @@ impl HttpJsonRpc {
...
+
```
Comment (paulhauner) [inline beacon_node/execution_layer/src/engine_api/http.rs]:
```suggestion
```

**Comment by pawanjay176** [comment]
I would assume passing the ef tests would require the proposer boosting implementation as well though?

File: testing/ef_tests/check_all_files_accessed.py [hunk 3]
```diff
@@ -74,6 +74,7 @@ def normalize_path(path):
             accessed_files += 1
 
 # Exit with an error if there were any files missed.
+print(missed)
```
Comment (paulhauner) [inline testing/ef_tests/check_all_files_accessed.py]:
```suggestion
```

I believe each missed file is already printed on [L71](https://github.com/sigp/lighthouse/blob/e19f0cdb49f46e0fd5485c10a4ddfa23c4fcb0c6/testing/ef_tests/check_all_files_accessed.py#L71).

**Comment by michaelsproul** [comment]
Ah, true. It would be nice to get Docker images though. I guess we could revive the Kintsugi Docker image builder? Alternatively we could mark the proposer boost tests as known failures and merge to `unstable`

**Review by paulhauner — CHANGES_REQUESTED** [review]
Looks good, thanks for this! I agree with @michaelsproul that we could merge this and then merge #2822 afterwards.

I got the EF tests passing (with some ignored tests) over here: https://github.com/paulhauner/lighthouse/commit/7fe54a6ed7c4bc1340c19132446e38596df711ff

**Comment by pawanjay176** [comment]
@paulhauner Done! You can merge when CI passes.

**Comment by michaelsproul** [comment]
bors retry

**Pull request closed without merging**